src/main/resources/xslt/preservica2schemaorg.xsl

```diff
@@ -220,38 +224,76 @@
+        <xsl:if test="./publisherRole ='channel_name'">
+          <xsl:value-of select="./publisher"/>
+        </xsl:if>
+      </xsl:for-each>
+    </xsl:variable>
+
+    <xsl:if test="pbcorePublisher">
+      <f:map key="publication">
+        <f:string key="@type">BroadcastEvent</f:string>
+        <!-- Define isLiveBroadcast from live extension field.  -->
+        <!-- TODO: Figure out what to do when live field isn't present in metadata. -->
+        <xsl:for-each select="/xip:DeliverableUnit/Metadata/pbc:PBCoreDescriptionDocument/pbcoreExtension/extension">
+          <xsl:if test="f:contains(., 'live:live') or f:contains(., 'live:ikke live')">
+            <f:boolean key="isLiveBroadcast">
+              <!-- Chooses between 'live' or 'ikke live' as these are boolean values.-->
+              <xsl:choose>
+                <xsl:when test="contains(., 'live:live')"><xsl:value-of select="f:true()"/></xsl:when>
+                <xsl:when test="contains(., 'live:ikke live')"><xsl:value-of select="false()"/></xsl:when>
+              </xsl:choose>
+            </f:boolean>
           </xsl:if>
-          <!-- TODO: Figure if it is possible to extract broadcaster in any meaningful way for the field 'broadcaster',
-                maybe from hovedgenre_id or kanalid. Otherwise it could be defined as 'Danmarks Radio'
-                for the first part of the project. -->
-        </f:map>
-      </xsl:if>
-    </xsl:for-each>
-    
+        </xsl:for-each>
+        <!-- Preservica contains two different fields for broadcaster-->
+        <xsl:if test="$publisherSpecific != ''">
+          <f:map key="publishedOn">
+            <f:string key="@type">BroadcastService</f:string>
+            <f:string key="broadcastDisplayName">
+              <xsl:value-of select="$publisherSpecific"/>
+            </f:string>
+            <xsl:if test="$publisherGeneral != ''"/>
+            <f:string key="alternateName">
+              <xsl:value-of select="$publisherGeneral"/>
+            </f:string>
+            <xsl:choose>
+              <xsl:when test="f:starts-with($publisherGeneral, 'dr')">
+                <f:map key="broadcaster">
+                  <f:string key="@type">Organization</f:string>
+                  <f:string key="legalName">DR</f:string>
+                </f:map>
+              </xsl:when>
+              <xsl:when test="f:starts-with($publisherGeneral, 'tv2')">
+                <f:map key="broadcaster">
+                  <f:string key="@type">Organization</f:string>
+                  <f:string key="legalName">TV 2 Danmark</f:string>
+                </f:map>
+              </xsl:when>
+              <xsl:when test="f:starts-with($publisherGeneral, 'tv3')">
+                <f:map key="broadcaster">
+                  <f:string key="@type">Organization</f:string>
+                  <f:string key="legalName">TV3</f:string>
+                </f:map>
+              </xsl:when>
+              <xsl:otherwise></xsl:otherwise>
```

Comment:
No `Unknown` here?

Reply:
In schema.org we don't really use unknowns. It's more common to omit the field then.